package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "install": "for d in packages/*/ ; do (cd \"$d\" && npm install); done",
+    "flow": "flow",
     "test": "jest",
```

Comment:
I think it would be better if we had three commands: `npm run flow`, `npm run jest` and `npm test` (which runs both of the previous commands). `npm test` is a standard so we should have all test-y things running inside of it (including `eslint` in the future).